src/main/java/games/strategy/engine/config/GameEnginePropertyReader.java

```diff
@@ -74,13 +75,17 @@ public String readLobbyPropertiesBackupFile() {
   public MaxMemorySetting readMaxMemory() {
     final String value = super.readProperty(GameEngineProperty.MAX_MEMORY);
 
-    if(Strings.nullToEmpty(value).isEmpty()) {
+    if (Strings.nullToEmpty(value).isEmpty()) {
       return MaxMemorySetting.NOT_SET;
     }
 
     return MaxMemorySetting.of(value);
   }
 
+  public boolean useExperimentalUI() {
```

Comment:
Checkstyle: "Abbreviation in name 'useExperimentalUI' must contain no more than '2' consecutive capital letters."